store/cachekv/store.go

```diff
@@ -180,12 +193,93 @@ func (store *Store) iterator(start, end []byte, ascending bool) types.Iterator {
 	}
 
 	store.dirtyItems(start, end)
-	cache = newMemIterator(start, end, store.sortedCache, &store.deleted, ascending)
+	cache = newMemIterator(start, end, store.sortedCache, store.deleted, ascending)
 
 	return newCacheMergeIterator(parent, cache, ascending)
 }
 
-// TODO(dudong2): need to bump up this func - (https://github.com/cosmos/cosmos-sdk/pull/10024)
+func findStartIndex(strL []string, startQ string) int {
```

Comment:
When I check this code's origin, it came from https://github.com/cosmos/cosmos-sdk/pull/10024. But the other test codes was not applied. Please check it.